exercises/two-fer/TwoFer.elm

```diff
@@ -0,0 +1,5 @@
+module TwoFer exposing (twoFer)
+
+
+twoFer name =
```

Comment:
I noticed you deleted the type annotation. Can you bring it back, please? The stub files should always show the type signature for the method so that the student will know what the function accepts and returns.